aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/util/ResourceModelHelper.java

```diff
@@ -91,4 +94,33 @@ public static boolean isReadReplicaPromotion(final ResourceModel previous, final
         return isDBClusterReadReplicaPromotion(previous, desired) ||
                 isDBInstanceReadReplicaPromotion(previous, desired);
     }
+
+    public static boolean shouldStartAutomaticBackupReplication(final ResourceModel previous, final ResourceModel desired) {
+        final String previousRegion = getAutomaticBackupReplicationRegion(previous);
+        final String desiredRegion = getAutomaticBackupReplicationRegion(desired);
+        return !StringUtils.isNullOrEmpty(desiredRegion) && !desiredRegion.equalsIgnoreCase(previousRegion);
+    }
+
+    public static boolean shouldStopAutomaticBackupReplication(final ResourceModel previous, final ResourceModel desired) {
+        final String previousRegion = getAutomaticBackupReplicationRegion(previous);
+        final String desiredRegion = getAutomaticBackupReplicationRegion(desired);
+        return !StringUtils.isNullOrEmpty(getAutomaticBackupReplicationRegion(previous)) && !previousRegion.equalsIgnoreCase(desiredRegion);
+    }
+
+    public static Integer getBackupRetentionPeriod(final ResourceModel model) {
```

Comment:
I believe we can return an unboxed value here, given there is a null-check later on